packages/nextra-theme-docs/src/constants.tsx

```diff
@@ -56,6 +60,44 @@ export const DEFAULT_THEME: DocsThemeConfig = {
     </span>
   ),
   bannerKey: 'nextra-banner',
+  notFoundLink() {
+    const config = useConfig()
+    const mounted = useMounted()
+    const { asPath } = useRouter()
+    return (
+      <Anchor
+        href={getGitIssueUrl({
+          repository: config.docsRepositoryBase,
+          title: `Found broken \`${mounted ? asPath : ''}\` link. Please fix!`,
+          labels: config.notFoundLabels
+        })}
+        newWindow
+      >
+        Submit an issue about broken link.
+      </Anchor>
+    )
+  },
```

Comment:
I think a need to move `<Anchor />` directly in `<NotFoundPage />`, same for `<Anchor />` in `serverSideErrorLink`